src/main/java/com/epam/edumanagementsystem/admin/rest/service/SubjectService.java

```diff
@@ -10,11 +10,11 @@ public interface SubjectService {
 
     List<Subject> findAll();
 
```

Comment:
Lets not return entity in service methods, this not a good practice. If there is a need to return a object after method execution it would be better to return DAO and not entity.